lib/std/compress/deflate/testdata/huffman-text.input

```diff
@@ -0,0 +1,13 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.

```

Comment:
Could we not include copyrighted test files?

Reply:
It helps to have the same exact test file as to be able to see any drift from the original Go implementation.
If this really is a blocker, tell me and I will remove the 2 impacted test.

The current implementation in master uses rfc1951.txt as a test file which is also [Copyrighted](https://github.com/ziglang/zig/blob/4731a6e5d57d5fe6c17c42028aebd9fce3682ddb/lib/std/compress/rfc1951.txt#L27).
The new one uses it too, so let me know.

Reply:
It would be slightly cleaner from a licensing perspective if it wasn't mixed like this, but it's fine. We're allowed to copy BSD-style licensed files into an MIT-licensed project.

Zig's readme says:

> This project redistributes code from other projects, some of which have other licenses besides MIT. Such licenses are generally similar to the MIT license for practical purposes. See the subdirectories and files inside lib/ for more details.

Please make sure test fixtures such as this are [excluded from zig installations](https://github.com/ziglang/zig/blob/7c4c49ff07de31b1b74f3dc26648ad4000113256/build.zig#L95-L102). (not for license reasons but for keeping installation size down)

Reply:
I believe we technically need to copy go's LICENSE file mentioned in the comment as well to include this file. The rfcs used as test cases have the full license text inline. I'm not sure this complication is worth it for this test case.

Reply:
Should be already excluded https://github.com/ziglang/zig/pull/10552/files#diff-f87bb3596894756629bc39d595fb18d479dc4edf168d93a911cadcb060f10fccR102

Reply:
I don't consider adding a file to be a big complication. Correct me if I'm wrong but the fact that this code is a derivative work of the Go code should mean the license file is useful for the whole package not just this test file.

Reply:
mmm ok I'm going to make an executive decision here. Omit this from the tests because the license says "all rights reserved". Let's just not deal with this problem.

Reply:
I replaced the test text with some zig code in `compress/deflate/testdata/huffman-text.input` but note that `git grep "All rights reserved"` returns a lot of matches from `lib/` and `deps/`.